src/absil/il.fs

```diff
@@ -2422,66 +2528,35 @@ let mk_ospec (typ:ILType,callconv,nm,genparams,formal_args,formal_ret) =
 
 let mkILGenericVirtualMethod (nm,access,genparams,actual_args,actual_ret,impl) = 
   { Name=nm;
+    Attributes= 
+      convertMemberAccess access |||
+      MethodAttributes.CheckAccessOnOverride ||| 
+      (match impl with MethodBody.Abstract -> MethodAttributes.Abstract ||| MethodAttributes.Virtual | _ -> MethodAttributes.Virtual);
+    ImplAttributes=MethodImplAttributes.Managed
     GenericParams=genparams;
     CallingConv=ILCallingConv.Instance;
-    mdKind=
-      MethodKind.Virtual 
-        { IsFinal=false; 
-          // REVIEW: We'll need to start setting this eventually
-          IsNewSlot = false;
-          IsCheckAccessOnOverride=true;
-          IsAbstract=(match impl with MethodBody.Abstract -> true | _ -> false) ; };
     Parameters=actual_args;
     Return=actual_ret;
-    Access=access;
     IsEntryPoint=false;
-    HasSecurity=false;
     SecurityDecls=emptyILSecurityDecls;
     CustomAttrs = emptyILCustomAttrs;
-    mdBody= mkMethBodyAux impl;
-    mdCodeKind=MethodCodeKind.IL;
-    IsInternalCall=false;
-    IsManaged=true;
-    IsForwardRef=false;
-    IsReqSecObj=false;
-    IsHideBySig=false;
-    IsSpecialName=false;
-    IsUnmanagedExport=false; 
-    IsSynchronized=false;
-    IsNoInline=false;
-    IsAggressiveInline=false
-    IsMustRun=false;
-    IsPreserveSig=false; }
+    mdBody= mkMethBodyAux impl; }
```

Comment:
semicolons - they can all go...

Reply:
please remove them all ;-)

Reply:
there are lot of semicolons even in code that i didn't changed in this file
are we sure we want to remove them all?

Reply:
Let's do it in a separate PR, that's ok, thanks